a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -229,17 +229,16 @@ def get_delete_load_balancer_flow(self, lb, deleteCompute):
 
         return (delete_LB_flow, store)
 
-    def get_new_LB_networking_subflow(self, topology):
+    def get_new_LB_networking_subflow(self, topology, network_type="flat"):
         """Create a sub-flow to setup networking.
 
         :returns: The flow to setup networking for a new amphora
         """
         LOG.info("Inside network subflow")
         new_LB_net_subflow = linear_flow.Flow(constants.
                                               LOADBALANCER_NETWORKING_SUBFLOW)
-        new_LB_net_subflow.add(a10_network_tasks.AllocateVIP(
-            requires=constants.LOADBALANCER,
-            provides=constants.VIP))
+        network_type = "vlan"
```

Comment:
Why we removed AllocateVIP ?? 
is network_type hardcoded for testing ?

Reply:
Yes it has been. Needs to be specified in the config file. Though i'm contemplating a better way of doing this.

Reply:
Yes it has been. Needs to be specified in the config file. Though i'm contemplating a better way of doing this.